gatsby-config.js

```diff
@@ -2,6 +2,18 @@
 /* eslint-disable camelcase */
 const path = require('path');
 
+let compareslug = '';
```

Comment:
```suggestion
let moduleSlug= '';
```
I'm not sure about the suggestion but could you please rename this variable?